src/jace/jax/api.py

```diff
@@ -0,0 +1,137 @@
+# JaCe - JAX Just-In-Time compilation using DaCe (Data Centric Parallel Programming)
+#
+# Copyright (c) 2024, ETH Zurich
+# All rights reserved.
+#
+# SPDX-License-Identifier: BSD-3-Clause
+
+"""Contains the implementation of the jit functioanlity of JaCe."""
+
+from __future__ import annotations
+
+import functools as ft
+from collections.abc import Callable, Mapping
+from typing import TYPE_CHECKING, Any, Literal, overload
+
+import jax as _jax_jax
```

Comment:
Other parts of the code use this `_jax` convention too.
I switched to it because it was indeed cryptic.
However, according to [PEP8](https://peps.python.org/pep-0008/#package-and-module-names), this is actually signalizes a C/C++ extension module.

Reply:
> However, according to [PEP8](https://peps.python.org/pep-0008/#package-and-module-names), this is actually signalizes a C/C++ extension module.

If you refer to the following paragraph of PEP8, I don't think is related:

> When an extension module written in C or C++ has an accompanying Python module that provides a higher level (e.g. more object oriented) interface, the C/C++ module has a leading underscore (e.g. _socket).

This only explains how a C extension module related to a python module should be named, but doesn't say that this is the only possible case to use an underscore in a module name or in an import alias for a 3rd party library: